.gitignore

```diff
@@ -4,3 +4,4 @@
 *.pyc
 config.py
 *.log
+.*.swp
```

Comment:
When opening the files in vim, it creates .swp files when the files are open.
So if someone commits while a file is open, then we have unnecessary files in the commit.